Jenkinsfile

```diff
@@ -852,12 +852,18 @@ pipeline {
                         label cachedCommitPragma(pragma: 'VM1-label', def_val: params.CI_UNIT_VM1_LABEL)
                     }
                     steps {
-                        scanRpms inst_repos: daosRepos(),
-                                 daos_pkg_version: daosPackagesVersion(next_version)
+                        runTest script: 'export DAOS_PKG_VERSION=' +
+                                        daosPackagesVersion(next_version) + '\n' +
+                                        'utils/scripts/helpers/scan_daos_maldet.sh',
+                                junit_files: 'maldetect_el8.xml',
+                                failure_artifacts: env.STAGE_NAME,
+                                ignore_failure: false,
+                                description: env.STAGE_NAME,
+                                context: 'test' + env.STAGE_NAME
```

Comment:
How does replacing `scanRpms` which does the node provisioning before calling `runTest` with just a `runTest` that does no node provisioning work?

I was curious to see how/if/why it worked so went looking for a Jenkins run with this stage enabled to see how it worked.  The most recent I could find was https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-9174/112/pipeline/744 on commit b8a6fb7.  But since that commit, the agent for this stage was changed from a Docker container to a VM, so surely it's now necessary to also provision a node, yes?  Probably want to switch back to `scanRpms` in that case.  But I'm also curious why the switch from Docker back to VM.  The run referenced above seems to indicate that it works in a Docker container, doesn't it?

In any case, I would consider it a requirement to be running this stage in your landing-requesting test runs in order for it to land.  Additionally, a pipeline-lib PR should be pushed to not skip the scanning stages once this does land.

Reply:
The last merge mangled the Jenkinsfile and I did not notice it.